src/thunderbots/software/ai/hl/stp/tactic/tactic.h

```diff
@@ -0,0 +1,17 @@
+#pragma once
+
+#include <memory>
+#include "ai/intent/intent.h"
+#include "ai/world/world.h"
+
+
+class Tactic
+{
+   public:
+    virtual std::unique_ptr<Intent> getNextIntent(const World& world,
```

Comment:
Comments would be nice here. EDIT: just noticed what you mentioned about comments

Reply:
added comments